config.py

```diff
@@ -191,6 +218,7 @@ def _validate_config(config: BaseConfig):
         assert config.WTF_CSRF_ENABLED
         assert not config.DEBUG
         assert not config.TESTING
+        assert config.LOG_LEVEL is not logging.DEBUG
```

Comment:
don't need this assertion? sometimes we might want to run production in DEBUG logging mode